pkg/cluster/cluster.go

```diff
@@ -362,25 +365,26 @@ func (c *Cluster) delete() {
 
 	pods, err := c.KubeCli.Pods(c.Namespace).List(option)
 	if err != nil {
-		panic(err)
+		return err
 	}
 	for i := range pods.Items {
 		if err := c.removePodAndService(pods.Items[i].Name); err != nil {
-			panic(err)
+			return err
 		}
 	}
 	if c.spec.Backup != nil {
 		err := c.bm.CleanupBackups()
 		if err != nil {
-			panic(err)
+			return fmt.Errorf("fail to cleanup backup: %v", err)
```

Comment:
this is not correct. we should continue to cleanup other stuff instead of simply stopping here.